internal/mode/static/nginx/config/servers_test.go

```diff
@@ -182,7 +187,7 @@ func TestExecuteServers(t *testing.T) {
 	)
 
 	gen := GeneratorImpl{}
-	results := gen.executeServers(conf, fakeGenerator)
+	results := gen.executeServers(conf, fakeGenerator, newKeepAliveChecker([]http.Upstream{}))
```

Comment:
You can also create a checker that can be reused throughout the tests. I think we do this with some validators throughout our test code:

```
alwaysFalseKeepAliveChecker := func(_ string) bool {return false}
alwaysTrueKeepAliveChecker := func(_ string) bool { return true}
```

I like this approach because it is obvious what the behavior of the checker is.